packages/devextreme/js/__internal/ui/overlay/m_overlay.ts

```diff
@@ -1225,26 +1366,30 @@ class Overlay<
   }
 
   _dispose(): void {
-    // @ts-expect-error
-    fx.stop(this._$content, false);
+    fx.stop(this._$content.get(0), false);
 
     this._toggleViewPortSubscription(false);
     this._toggleSubscriptions(false);
     this._updateZIndexStackPosition(false);
 
-    this._actions = null;
-    this._parentsScrollSubscriptionInfo = null;
+    this._actions = undefined;
+    this._parentsScrollSubscriptionInfo = undefined;
 
     super._dispose();
 
     this._toggleSafariScrolling();
-    this.option('visible') && zIndexPool.remove(this._zIndex);
+
+    if (this.option('visible')) {
```

Comment:
safer to extract visible option first
```suggestion
     const { visible } = this.option();

    if (visible) {
```

Reply:
eca8905e08c0af1d0eedada4c1e167e77335af2c, 54fb3761139b89233aabdcf60cfc00701960f9b6, 331517e56476f40cbf591e671dcd6b8e43edb8ca, 38d39427a4658cbf6852f4fe72f979b6c9c54a49